docker/Dockerfile

```diff
@@ -41,15 +42,15 @@ RUN apt-get update && \
         update-alternatives --install /usr/bin/clang-tidy    clang-tidy    /usr/bin/clang-tidy-${CLANG_VERSION} 999 && \
         update-alternatives --install /usr/bin/clang-format  clang-format  /usr/bin/clang-format-${CLANG_VERSION} 999
 
-ENV GEODE_VERSION 1.9.0
+ENV GEODE_VERSION 1.12.0
 RUN wget "https://www.apache.org/dyn/closer.cgi?action=download&filename=geode/${GEODE_VERSION}/apache-geode-${GEODE_VERSION}.tgz" --quiet -O - | \
         tar xzf -
 
 ENV RAT_VERSION 0.13
 RUN wget "https://www.apache.org/dyn/closer.cgi?action=download&filename=creadur/apache-rat-${RAT_VERSION}/apache-rat-${RAT_VERSION}-bin.tar.gz" --quiet -O - | \
         tar xzf -
 
-ENV CMAKE_VERSION 3.12.2
+ENV CMAKE_VERSION 3.12.4
```

Comment:
Might as well take it to 3.17 or whatever the latest is today.

Reply:
I stuck with 3.12 since that is min version we specify in the CMakeLists.txt.  It seemed worth keeping that in sync but maybe it doesn't matter?